components/pagination/pagination.ts

```diff
@@ -0,0 +1,163 @@
+import { Component, Input, Output, OnChanges, SimpleChanges, EventEmitter } from "@angular/core";
+
+@Component({
+    selector: "sui-pagination",
+    template: `
+<div class="ui pagination menu">
+    <a *ngIf="showBoundary" class="item"  (click)="selectPage(1)" [ngClass]="{'disabled': page===1}">
+        <i class="fast backward icon"></i>
```

Comment:
Not sure what that will do to font sizing though... Icons may be a better bet - if so, use the icon versions of those characters? I think the vertical lines in the icons are a bit confusing